include/oxen/quic/loop.hpp

```diff
@@ -22,6 +22,45 @@ namespace oxen::quic
 
     static void setup_libevent_logging();
 
+    using loop_time = std::chrono::microseconds;
```

Comment:
Ditch this typedef: the original is a more descriptive type to have in the exposed API than the typedef.